internal/config/config.go

```diff
@@ -95,6 +96,9 @@ func setDefaults(cfg *Config) {
 	if cfg.RunStateFile == "" {
 		cfg.RunStateFile = filepath.Join(cfg.RunDir, "exod.json")
 	}
+	if cfg.TokenFile == "" {
```

Comment:
If it can have multiple tokens, should it be called "TokensFile" (plural)?

I'm not clear on how rotation will work & if you need token metadata. Maybe we should practice YAGNI for that & just support one token at a time, or actually think through what is needed for rotation.

Reply:
Renamed. I don't think the rotation story matters much right now. I added a very simple implementation of it just because it's been my experience that it's one of the rare features that if you only think about it only once you need it, it's too late. But I don't think that any of that is relevant to this very simple use case. Happy to remove entirely if you'd prefer.